src/main/resources/org/jenkinsci/plugins/periodicbackup/FullBackup/help-includesString.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    A list of all includes in Ant syntax; separated by ';'.
```

Comment:
can we mention it is optional?

Reply:
Done! I've added clarifications here and in the `excludedString.html` file.